configs/ipq40xx-generic.config

```diff
@@ -0,0 +1,16 @@
+CONFIG_TARGET_ipq40xx=y
+CONFIG_TARGET_ipq40xx_generic=y
+CONFIG_TARGET_ipq40xx_DEVICE_glinet_gl-b1300=y
+CONFIG_TARGET_BOARD="ipq40xx"
+CONFIG_TARGET_ROOTFS_INITRAMFS=y
+CONFIG_TARGET_INITRAMFS_COMPRESSION_NONE=y
+CONFIG_SDK=y
+COBFIG_PACKAGE_ath10k-firmware-qca4019-ct
+CONFIG_PACKAGE_kmod-ath10k-ct=m
+CONFIG_PACKAGE_kmod-usb-dwc3=m
+CONFIG_PACKAGE_kmod-usb3=y
+CONFIG_PACKAGE_kmod-usb-dwc3-of-simple=y
+CONFIG_TARGET_ARCH_PACKAGES="arm_cortex-a7_neon-vfpv4"
+CONFIG_PACKAGE_wireless-regdb=m
```

Comment:
is this really needed?  This package should be pulled be the wifi-drivers as dependency

Reply:
i dosn't found his packages in the common so i add it to the ipq40xx specific because there are also in the openwrt config for ipq40xx Target. The only packages there are actually not needed are the fritz packages.